docs/organizations/org-slugs-in-urls.mdx

```diff
@@ -131,15 +131,15 @@ This guide shows you how to add organization slugs to your app's URLs, configure
   ## Configure `clerkMiddleware()` to set the active organization
 
   > [!TIP]
-  > If your app doesn't use `clerkMiddleware()`, or you prefer to manually set the active organization, use the [`setActive()`](/docs/references/javascript/clerk) method to control the active organization on the client-side. See [this guide](/docs/organizations/force-organizations#set-an-active-organization-based-on-the-url) to learn how to manually activate a specific organization based on the URL.
+  > If your app doesn't use `clerkMiddleware()`, or you prefer to manually set the active organization, use the [`setActive()`](/docs/references/javascript/clerk) method to control the active organization on the client side.
 
-  With [`clerkMiddleware()`](/docs/references/nextjs/clerk-middleware), you can use the [`organizationSyncOptions`](/docs/references/nextjs/clerk-middleware#organization-sync-options) property to declare URL patterns that determine whether a specific organization or user's personal account should be activated.
+  With [`clerkMiddleware()`](/docs/references/nextjs/clerk-middleware), you can use the [`organizationSyncOptions`](/docs/references/nextjs/clerk-middleware#organization-sync-options) property to declare URL patterns that determine whether a specific organization or a user's personal workspace should be activated.
 
   If the middleware detects one of these patterns in the URL and finds that a different organization is active in the session, it'll attempt to set the specified organization as the active one.
 
   In the following example, two `organizationPatterns` are defined: one for the root (e.g., `/orgs/acmecorp`) and one as the wildcard matcher `(.*)` to match `/orgs/acmecorp/any/other/resource`. This configuration ensures that the path `/orgs/:slug` with any optional trailing path segments will set the organization indicated by the slug as the active one.
 
-  The same pattern is used with `personalAccountPatterns` to match the user's personal account.
+  The same pattern is used with `personalWorkspacePatterns` to match the user's personal workspace.
```

Comment:
This is going to be released soon: https://github.com/clerk/javascript/pull/6401